modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
+* stable-{product-version}
+
+RC release versions are only available in candidate-{product-version}.
+
+GA release versions are only availble in fast-{product-version} or stable-{product-version}.
+
+More details about each channel and their usage are in the following sections.
+
+= candidate-{product-version}
+
+The candidate channel allows upgrading from an RC version to a later RC version and
+upgrading from a previous minor version of {product-title} to the current RC version.
+You cannot upgrade from an RC release to a GA release of {product-title}.
 
 = fast-{product-version}
 
-The fast channel is updated with new {product-version} patch versions as soon as Red
-Hat declares they are generally available. Use this channel if you wish to receive
-updates as soon as they are available or for your pre-production environments when
-participating in the connected customer program. This channel will contain all
-z-stream ({product-version}.z) updates but will not suggest upgrades to the next
-minor release (4.5.z) when the next minor release is available.
+The fast channel, containing only GA releases, is updated with new
+{product-version} patch versions as soon as Red Hat declares they are generally
+available. Use this channel if you wish to receive updates as soon as they are
+available or for your pre-production environments when participating in the connected
+customer program. This channel will contain all z-stream ({product-version}.z)
+updates but will not suggest upgrades to the next minor release (4.5.z) when the next
+minor release is available. This channel also includes the upgrade from the previous
+4.3 release.
 
 = stable-{product-version}
 
-The stable channel will contain updates on a time delay as they are gradually
-rolled out to customers based on data from our SRE teams, support services, and
-pre-production and production environments that participate in our connected
-customer program, rather than being immediately available as they are in the fast
-channel. For patch and CVE fixes this can range from several hours to a day and
-allows an extra period of assessment in how the software performs. If issues are
-detected during rollout, upgrades to that version may be blocked in both the fast
-and stable channels, and a new version may be introduced that will be the new
-preferred upgrade target.
+The stable channel, containing only GA releases, will contain updates on a time delay
+as they are gradually rolled out to customers based on data from our SRE teams,
+support services, and pre-production and production environments that participate in
+our connected customer program, rather than being immediately available as they are
+in the fast channel. For patch and CVE fixes this can range from several hours to a
+day and allows an extra period of assessment in how the software performs. This
+channel also includes the upgrade from the previous 4.3 release.
+
+= Fast and Stable Channel Usage and Strategies
```

Comment:
This section should go below the *Upgrade Version Paths* section, because you need to understand how updates are supposed to be handled before you can decide between fast/stable.  The content above this about the channels themselves only talks about relesases, and hasn't yet said anything about updates.